marshmallow_utils/fields/localizededtfstring.py

```diff
@@ -0,0 +1,40 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2016-2020 CERN.
+#
+# Marshmallow-Utils is free software; you can redistribute it and/or modify
+# it under the terms of the MIT License; see LICENSE file for more details.
+
+"""Localized Extended Date(/Time) Format Level 0 date string field."""
+
+import calendar
+from datetime import date, datetime
+
+from babel_edtf import format_edtf
+from marshmallow import fields
+
+
+class LocalizedEDTFString(fields.String):
+    """Localized Extended Date(/Time) Format Level 0 date string field.
+
+    Useful only for dumping.
+    """
+
+    def __init__(self, format=None, locale=None, **kwargs):
+        """Constructor.
+
+        :param format: The format to use.
+        :param locale: The current locale or a callable returning the current
+                       locale.
+        """
+        self._format = format
+        self._locale = locale
+        kwargs.setdefault('dump_only', True)
+        super(LocalizedEDTFString, self).__init__(**kwargs)
+
+    def _serialize(self, value, attr, data, **kwargs):
+        """Serialize a localized EDTF level 0 formatted date string."""
+        locale = self._locale() if callable(self._locale) else self._locale
```

Comment:
Question: Wouldn't this end up in a call to `format_edtf` with `locale=None` (which removes the default `LC_TIME` - https://github.com/inveniosoftware/babel-edtf/blob/master/babel_edtf/__init__.py#L98).

It won't fail, since `Locale.parse(locale)` simply returns `None`. But it will omit the default (Maybe it's intended 🤔 )

Same happens for `format`, though when `skeleton=None` the inner default for parsing seems to be `medium` too.

Reply:
Yes, correct. Good catch! :-)

Reply:
Fixed